queenbee/io/__init__.py

```diff
@@ -30,9 +30,9 @@ class IOBase(BaseModel):
     IOBase is used within Operators, Recipes and Workflows.
     """
 
-    parameters: List[IOItem]
+    inputs: List[IOItem]
```

Comment:
I like this List[IOItem] instead of List[Union[xxx,xxx,xxx,xxx,xxx,xxx,xxx,xxx]]

Reply:
We can't do that anymore! This code will also change! 😀

This is now similar to Radiance materials. We have to handle all the different types. JSONSchema/OpenAPI doesn't support check for subclasses. It should be the exact object type.